### DISCUSSION TITLE
RaspberryMatic rpi3 -neue kernel 20231024, 20231030,

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#2493] [🇩🇪 Generelle Diskussionen]
Hat jemand von euch festgestellt, dass nach der Installation von snapschoot 28.10.23 auf rpi3 kein Booten mehr möglich ist? Die rote LED am rpi3 leuchtet ständig, als ob das Gerät ständig ausgeschaltet wäre, Ich kann zur Überprüfung keinen Monitor anschließen, Kann es jemand überprüfen? entweder bestätigen oder dementieren :-)

**Comment by ghost** [comment]
ok – ich flashe die Karte auf RaspberryMatic-3.71.12.20231024-7b54e5-rpi3 – wenn es später funktioniert, gehe ich zu: RaspberryMatic-3.71.12.20231026-6d7d47-rpi3 und schließlich versuche ich es noch einmal: RaspberryMatic-3.71 .12.20231028 -57afd5-rpi3

**Comment by jens-maus** [comment]
Kann gut sein das der heutige Snapshot ein Problem hat weil da nen neuerer RaspberryPi Kernel mit drin ist den ich noch nicht selbst getestet habe... Tester mit entsprechendem Feedback willkommen!

**Comment by ghost** [comment]
Ja, ich wusste, dass es einen neuen Kernel gibt und ich wollte ihn testen :-) Ich habe die Karte mit dem neuen Kernel geflasht: RaspberryMatic-3.71.12.20231028-rpi3.img um auszuschließen, dass etwas mit meinem System nicht stimmt (frisches System ohne Backup) - und es ist das gleiche: das System bootet nicht, ich werde jetzt RaspberryMatic-3.71.12.20231026-6d7d47-rpi3.img ausprobieren und ich werde es euch wissen lassen , um völlig auszuschließen, dass es mein System ist, das auf diese Weise reagiert

**Reply by ghost** [reply]
Ich kann bestätigen, dass das System mit dem alten Kernel : RaspberryMatic-3.71.12.20231026-rpi3.img normal startet. Das heißt, der neue 20231024 Kernel bringt Änderungen mit sich, die mir nicht bekannt sind. Und ich habe nicht das Wissen, um es zu lösen, aber Sie sind:  @jens-maus , @Baxxy13 , @jp112sdl usw. und für dich wird es sicher ein Kinderspiel sein ;-))))

**Reply by Baxxy13** [reply]
Ich bleibe noch eine Woche in der 🌞 
Kann also nichts testen.

**Reply by jens-maus** [reply]
Alles kein Problem. Genies die Sonne und entspann dich. Wir reden hier nur von einem developerment nightly snapshot. der darf ruhig mal ne weile kaputt sein. Nightly Snapshosts sind ja mit Absicht nichts für produktive Umgebungen....

**Reply by ghost** [reply]
im neuen Snapshot RaspberryMatic-3.71.12.20231031-a7ca31-rpi3  startet das System auch nicht (neuer Kernel und das gleiche Problem)

**Reply by jens-maus** [reply]
Ja, schon bemerkt und hab auch erst einmal keine Lösung dafür parat leider. Hier muss ich erst einmal auf einen älteren kernel zurückgehen bis ich den Grund gefunden habe warum das System mit den neueren Kernelversionen nicht mehr geht. Scheint wohl am geänderten DeviceTreeOverlay zu liegen der einige Änderungen mit sich bringt und deshalb der U-boot bootloader nicht mehr funktionieren will.
